Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -27,6 +27,14 @@ struct PreplannedListItemView: View {
     /// A Boolean value indicating whether the metadata view is presented.
     @State private var metadataViewIsPresented = false
     
+    /// The download state of the preplanned map model.
+    enum DownloadState {
```

Comment:
This should still be private.

```suggestion
    fileprivate enum DownloadState {
```